src/CatBox.NET/CatBoxConfig.cs

```diff
@@ -8,10 +8,10 @@ public record CatBoxConfig
     /// <summary>
     /// URL for the catbox.moe domain
     /// </summary>
-    public Uri CatBoxUrl { get; set; }
+    public Uri? CatBoxUrl { get; set; }
```

Comment:
What's the purpose of setting `CatBoxUrl` and `LitterboxUrl` to be nullable here? They technically don't have to be set if the user is using the other `Client`, but they are required for their respective clients

Reply:
It was basically to appease Visual Studio IntelliSense as I did not want to default initialize them.

Reply:
Yeah, that makes sense